### DISCUSSION TITLE
Error typescript if passe param in use query

### DISCUSSION THREAD

**Discussion by kamalkech** [trpc/trpc#2622] [Q&A] [answered]
in server 
![image](https://user-images.githubusercontent.com/1313428/188126265-677d3c09-b9b2-417f-b36c-0be667f10eb8.png)

in client component to get post by id
![image](https://user-images.githubusercontent.com/1313428/188126519-7c6911e2-e26e-4169-975f-5c28cc939d3a.png)

any idea ?

**Comment by KATT** [comment, marked as answer]
Have you enabled strict mode? https://trpc.io/docs/v9/nextjs#2-enable-strict-mode

**Reply by kamalkech** [reply]
yes i already enabled by still get this error !!

![image](https://user-images.githubusercontent.com/1313428/188128973-ad545df1-82aa-4026-b47e-1f19d22666ee.png)

**Reply by KATT** [reply]
Alright, then we need a reproduction in order to help you unfortunately

**Reply by kamalkech** [reply]
u can found this app here https://github.com/kamalkech/nextjs-awesome
and let me know what do u think !
may be an error in last version or zod ??

**Reply by kamalkech** [reply]
i fixed by update trcp.ts in folder utils.
